src/QA/SoftMocks.php

```diff
@@ -94,6 +94,15 @@ protected function p(\PhpParser\Node $node)
         return $prefix . $this->{'p' . $node->getType()}($node);
     }
 
+    public function __call($name, $args)
```

Comment:
I don't think that __call is a proper way to fix the problem, to be honest. I will look into why you get this error a bit later, thanks for report!